diff -durN mangoquest-0.6.1.orig/configure.in mangoquest-0.6.1/configure.in
--- mangoquest-0.6.1.orig/configure.in	Fri Aug  3 22:09:04 2001
+++ mangoquest-0.6.1/configure.in	Tue Aug  7 10:05:23 2001
@@ -31,8 +31,8 @@
 dnl General options
 dnl --------------------------------------------------------------------------
 TR_CPPFLAGS=""
-TR_CFLAGS="-O2 -Wall -fomit-frame-pointer -ffast-math -fexpensive-optimizations"
-TR_CXXFLAGS="-O2 -Wall -fomit-frame-pointer -ffast-math -fexpensive-optimizations"
+TR_CFLAGS="-Wall -fomit-frame-pointer -ffast-math -fexpensive-optimizations"
+TR_CXXFLAGS="-Wall -fomit-frame-pointer -ffast-math -fexpensive-optimizations"
 TR_LIBS=""
 
 case "$host" in
