diff -durN mangoquest-0.6.1.orig/doc/Makefile.am mangoquest-0.6.1/doc/Makefile.am
--- mangoquest-0.6.1.orig/doc/Makefile.am	Fri Aug  3 22:09:04 2001
+++ mangoquest-0.6.1/doc/Makefile.am	Tue Aug  7 10:48:18 2001
@@ -3,15 +3,15 @@
 SHELL=@SHELL@
 
 install:
-	$(SHELL) ./../mkinstalldirs $(pkgdatadir)/
-	$(SHELL) ./../mkinstalldirs $(pkgdatadir)/doc
-	cp -r manuals.tar.bz2 $(pkgdatadir)/doc/
-	chmod -R 755 $(pkgdatadir)
+	$(SHELL) ./../mkinstalldirs $(DESTDIR)$(pkgdatadir)/
+	$(SHELL) ./../mkinstalldirs $(DESTDIR)$(pkgdatadir)/doc
+	cp -r manuals.tar.bz2 $(DESTDIR)$(pkgdatadir)/doc/
+	chmod -R 755 $(DESTDIR)$(pkgdatadir)
 
 # Directories in EXTRA_DIST doen't work?
 dist-hook:
-	#cp -r -p schema.fig.gz $(distdir)
+	#cp -r -p schema.fig.gz $(DESTDIR)$(distdir)
 
 uninstall:
-	@echo "Removing $(pkgdatadir) ..."
-	rm -rf $(pkgdatadir)
+	@echo "Removing $(DESTDIR)$(pkgdatadir) ..."
+	rm -rf $(DESTDIR)$(pkgdatadir)
diff -durN mangoquest-0.6.1.orig/game_data/Makefile.am mangoquest-0.6.1/game_data/Makefile.am
--- mangoquest-0.6.1.orig/game_data/Makefile.am	Fri Aug  3 22:09:04 2001
+++ mangoquest-0.6.1/game_data/Makefile.am	Tue Aug  7 10:45:42 2001
@@ -3,19 +3,19 @@
 SHELL=@SHELL@
 
 install:
-	$(SHELL) ./../mkinstalldirs $(pkgdatadir)
-	cp -r fonts hud sons textures maps $(pkgdatadir)
-	chmod -R 755 $(pkgdatadir)
+	$(SHELL) ./../mkinstalldirs $(DESTDIR)$(pkgdatadir)
+	cp -r fonts hud sons textures maps $(DESTDIR)$(pkgdatadir)
+	chmod -R 755 $(DESTDIR)$(pkgdatadir)
 
 # Directories in EXTRA_DIST doen't work?
 dist-hook:
-	cp -r -p fonts $(distdir)
-	cp -r -p hud  $(distdir)
-	cp -r -p sons  $(distdir)
-	cp -r -p textures  $(distdir)
-	cp -r -p maps  $(distdir)
-	cp -r -p $(distdir)
+	cp -r -p fonts $(DESTDIR)$(distdir)
+	cp -r -p hud  $(DESTDIR)$(distdir)
+	cp -r -p sons  $(DESTDIR)$(distdir)
+	cp -r -p textures  $(DESTDIR)$(distdir)
+	cp -r -p maps  $(DESTDIR)$(distdir)
+	cp -r -p $(DESTDIR)$(distdir)
 
 uninstall:
-	@echo "Removing $(pkgdatadir) ..."
-	rm -rf $(pkgdatadir)
+	@echo "Removing $(DESTDIR)$(pkgdatadir) ..."
+	rm -rf $(DESTDIR)$(pkgdatadir)
diff -durN mangoquest-0.6.1.orig/mangopeeler/doc/Makefile.am mangoquest-0.6.1/mangopeeler/doc/Makefile.am
--- mangoquest-0.6.1.orig/mangopeeler/doc/Makefile.am	Fri Aug  3 22:09:04 2001
+++ mangoquest-0.6.1/mangopeeler/doc/Makefile.am	Tue Aug  7 10:46:18 2001
@@ -4,12 +4,12 @@
 
 
 install:
-	$(SHELL) ./../mkinstalldirs $(pkgdatadir)
-	chmod -R 755 $(pkgdatadir)
+	$(SHELL) ./../mkinstalldirs $(DESTDIR)$(pkgdatadir)
+	chmod -R 755 $(DESTDIR)$(pkgdatadir)
 
 # Directories in EXTRA_DIST doen't work?
 dist-hook:
 
 uninstall:
-	@echo "Removing $(pkgdatadir) ..."
-	rm -rf $(pkgdatadir)
+	@echo "Removing $(DESTDIR)$(pkgdatadir) ..."
+	rm -rf $(DESTDIR)$(pkgdatadir)
diff -durN mangoquest-0.6.1.orig/mangopeeler/editor_data/Makefile.am mangoquest-0.6.1/mangopeeler/editor_data/Makefile.am
--- mangoquest-0.6.1.orig/mangopeeler/editor_data/Makefile.am	Fri Aug  3 22:09:04 2001
+++ mangoquest-0.6.1/mangopeeler/editor_data/Makefile.am	Tue Aug  7 10:46:47 2001
@@ -3,19 +3,19 @@
 SHELL=@SHELL@
 
 install:
-	$(SHELL) ./../mkinstalldirs $(pkgdatadir)
-	cp -r bonus fonts interface autres icone.bmp splash.png  $(pkgdatadir)/
-	chmod -R 755 $(pkgdatadir)
+	$(SHELL) ./../mkinstalldirs $(DESTDIR)$(pkgdatadir)
+	cp -r bonus fonts interface autres icone.bmp splash.png  $(DESTDIR)$(pkgdatadir)/
+	chmod -R 755 $(DESTDIR)$(pkgdatadir)
 
 # Directories in EXTRA_DIST doen't work?
 dist-hook:
-	cp -r -p bonus $(distdir)
-	cp -r -p fonts  $(distdir)
-	cp -r -p interface  $(distdir)
-	cp -r -p icone.bmp  $(distdir)
-	cp -r -p splash.png  $(distdir)
-	cp -r -p autres  $(distdir)
+	cp -r -p bonus $(DESTDIR)$(distdir)
+	cp -r -p fonts  $(DESTDIR)$(distdir)
+	cp -r -p interface  $(DESTDIR)$(distdir)
+	cp -r -p icone.bmp  $(DESTDIR)$(distdir)
+	cp -r -p splash.png  $(DESTDIR)$(distdir)
+	cp -r -p autres  $(DESTDIR)$(distdir)
 
 uninstall:
-	@echo "Removing $(pkgdatadir) ..."
-	rm -rf $(pkgdatadir)
+	@echo "Removing $(DESTDIR)$(pkgdatadir) ..."
+	rm -rf $(DESTDIR)$(pkgdatadir)
